src/Router.ts

```diff
@@ -201,13 +250,15 @@ export class Router<C extends Context> extends Evented {
 	private _history?: History;
 	private _routes: Route<Context, Parameters>[];
 	private _started?: boolean;
+	private _outletStack: Map<string, any> = new Map<string, OutletContext>();
```

Comment:
A nit, but a stack would be a `Set` or an array where you could push and pop...  This is some sort of map.